options/options_settable_test.cc

```diff
@@ -388,6 +388,7 @@ TEST_F(OptionsSettableTest, ColumnFamilyOptionsAllFieldsSettable) {
        sizeof(std::shared_ptr<MemTableRepFactory>)},
       {offset_of(&ColumnFamilyOptions::table_properties_collector_factories),
        sizeof(ColumnFamilyOptions::TablePropertiesCollectorFactories)},
+      {offset_of(&ColumnFamilyOptions::blob_path), sizeof(std::string)},
```

Comment:
I do not think this option should be in the skip list but should be explicitly set by this test.